src/editors/editorsController.ts

```diff
@@ -27,7 +28,8 @@ export default class EditorsController {
     log.info('activating...');
```

Comment:
Where does logger write? into file? Or debug window?

Reply:
Currently these are going into an output channel called `mongodb`. In the extension workspace  the channel is a bit of a secret in the output panel with a dropdown:
<img width="1442" alt="Screen Shot 2020-03-08 at 8 06 34 PM" src="https://user-images.githubusercontent.com/1791149/76169444-7b404200-6178-11ea-9a32-62ffc7ec8011.png">

Reply:
Ah here it is! Really hidden :)